src/main/java/com/github/dockerjava/api/model/Node.java

```diff
@@ -8,20 +9,30 @@
  * A node as returned by the /events API, for instance, when Swarm is used.
  */
 @JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Node {
 
     @JsonProperty("Name")
     private String name;
 
-    @JsonProperty("Id")
+    @JsonProperty("ID")
```

Comment:
So Looks like 'Node' is being used in [Event](https://github.com/docker-java/docker-java/blob/master/src/main/java/com/github/dockerjava/api/model/Event.java#L24) response.

But I can't seem to find where event response sends a node. I looks at [1.23](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.23/) and [1.18](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.18/) none of those events have `Node`.  

I am confused 1. why its there 2. what kind of backward compatibility this would be breaking.